camera/CMakeLists.txt

```diff
@@ -1,7 +1,65 @@
 cmake_minimum_required(VERSION 3.2)
 project(camera)
+set(ORESATLIVESW_VERSION_MAJOR "0")
+set(ORESATLIVESW_VERSION_MINOR "0")
+set(ORESATLIVESW_VERSION_PATCH "0")
+set(ORESATLIVESW_VERSION_REVISION "1")
+string(CONCAT CPACK_PACKAGE_VERSION
+    "${ORESATLIVESW_VERSION_MAJOR}."
+    "${ORESATLIVESW_VERSION_MINOR}."
+    "${ORESATLIVESW_VERSION_PATCH}-"
+    "${ORESATLIVESW_VERSION_REVISION}"
+    )
+# Include and output directories
+include_directories(
+    ${PROJECT_SOURCE_DIR}
+)
+set(ORESATLIVESW_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/bin/${CMAKE_BUILD_TYPE})
+set(ORESATLIVESW_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/lib/${CMAKE_BUILD_TYPE})
+set(ORESATLIVESW_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/lib/${CMAKE_BUILD_TYPE})
```

Comment:
Are `ORESATLIVESW_ARCHIVE_OUTPUT_DIRECTORY` and `ORESATLIVESW_LIBRARY_OUTPUT_DIRECTORY` just boilerplate in case we need them later?

Is there something we should put in or near the `install()` command that installs libraries if any get created?

Reply:
they are not currently used, so I'll remove them.